File: random_test.m
seed=1;
RandStream.setGlobalStream(RandStream('mt19937ar','seed',seed));

randn(zeros(1,5),eye(5))

random_nums = rand(1,5);

seed=1;
RandStream.setGlobalStream(RandStream('mt19937ar','seed',seed));

random_mvnrnd = normrnd(0,0.5*varxi,Nproducts,1);

File: random_test.py
# -*- coding: utf-8 -*-
"""
Created on Fri Jun 29 14:18:36 2018

@author: L1YAK01
"""

import numpy as np
import random

np.random.seed(1)
random_nums = np.random.random(5)

Nproducts = 2
varxi= 4 #temporary
np.random.seed(1)
random_mvn = np.random.normal(0, 0.5*varxi, Nproducts)

random.seed(1)
rand = random.gauss([0]*5,np.identity(5))